### DISCUSSION TITLE
foreground/firstaid/FirstAid

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#900] [Announcements]
# foreground/firstaid/FirstAid

项目名称（uni-FirstAid ） pda版本也会涉及到急诊pda设置广播动作和广播标签等.钉钉小程序，每次发布需要管理人登陆钉钉开发者后台发布 分支说明： 项目名称（uni-firstaid-app） 急救移动端包含 APK 钉钉小程序 移动端在pc显示 企业微信 急救移动端线上服务API地址 修改请求地址从 config/env-dev.js中...

http://localhost:8080/foreground/firstaid/FirstAid.html

**Comment by Kevin-Jii** [comment]
分支有改动记得即使更新

**Reply by Kevin-Jii** [reply]
测
